src/qz/printer/PrintServiceMatcher.java

```diff
@@ -84,6 +98,8 @@ public static NativePrinter matchPrinter(String printerSearch, boolean silent) {
 
         if (!silent) { log.debug("Searching for PrintService matching {}", printerSearch); }
 
+        // Fix for https://github.com/qzind/tray/issues/931
+        // This is more than an optimization, removal will lead to a regression
```

Comment:
This comment does not appear to reference any code changes.  Is it outdated now?

Reply:
yea we can ditch it. it was for me, because my first pass i broke this code, thinking the getdefault was just an optimization.

Reply:
Oh whoops, wrong bug report.  Yeah it can stay!